packages/@coorpacademy-components/src/template/slides-review/index.js

```diff
@@ -0,0 +1,418 @@
+import React, {useState, useReducer, useMemo} from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import getOr from 'lodash/fp/getOr';
+import omit from 'lodash/fp/omit';
+import isNil from 'lodash/fp/isNil';
+import ReviewBackground from '../../atom/review-background';
+import {ICON_VALUES} from '../../atom/review-header-step-item';
+import ReviewHeader from '../../organism/review-header';
+import ReviewCorrectionPopin from '../../molecule/review-correction-popin';
+import Answer from '../../molecule/answer';
+import ButtonLink from '../../atom/button-link';
+import style from './style.css';
+
+const stylesByPosition = {
+  0: style.position0,
+  1: style.position1,
+  2: style.position2,
+  3: style.position3,
+  4: style.position4
+};
```

Comment:
as styles are assigned _on the go_  for the slides, an object has the styles pre-charged